[CHANGELOG] v5.0.0-alpha.13 (#23115)
mui-org/material-ui: CHANGELOG.md

### [Versions](https://material-ui.com/versions/)

## 5.0.0-alpha.13

###### _Oct 17 2020_

Big thanks to the 25 contributors who made this release possible.
Here are some highlights ✨:

- 📦 Ship modern bundle (#22814) @eps1lon.

>   This is a significant update to the [supported browsers](https://next.material-ui.com/getting-started/supported-platforms/) of Material-UI.

Reviewer: ```suggestion
  This is a significant update to the [browsers supported by Material-UI](https://next.material-ui.com/getting-started/supported-platforms/).
```

>   The previous policy was defined 2 years ago, the landscape has evolved during the period. The package now includes 4 bundles:

Reviewer: ```suggestion
  The previous policy was defined 2 years ago, and the landscape has evolved since then. The package now includes 4 bundles:
```

  1. `stable` (default, formerly `esm`) which targets a snapshot (on release) of `> 0.5%, last 2 versions, Firefox ESR, not dead, not IE 11"`
  2. `node` (formerly default) which targets a snapshot (on release) of `maintained node versions`
  3. `legacy` (new) which is `stable` + IE11

>   4. `modern` (formerly `es`) which targets the last 1 version of evergreen browsers and active node (currently that is 14.

Author: ```suggestion
  4. `modern` (formerly `es`) which targets the last 1 version of evergreen browsers and active node (currently that is 14
```

>   The change yields a -6% reduction of bundle size 📦 (Babel only).

Reviewer: ```suggestion
  The change yields a 6% reduction in bundle size 📦 (Babel only).
```

>   In the coming weeks, we will refactor the internals to leverage the new browser's capability that this drop of older platforms allows. For instance, we might be able to remove the span we render inside the `<Button>` to work around [Flexbug #9](https://github.com/philipwalton/flexbugs/blob/master/README.md#flexbug-9).

Reviewer: ```suggestion
  In the coming weeks, we will refactor the internals to take advantage of the new browser capabilities that dropping these older platforms allows. For instance, we might be able to remove the span we render inside the `<Button>` to work around [Flexbug #9](https://github.com/philipwalton/flexbugs/blob/master/README.md#flexbug-9).
```

>   You can find the new [Supported platforms documentation](https://next.material-ui.com/getting-started/supported-platforms/) and [new "minimizing bundle size" guide](https://next.material-ui.com/guides/minimizing-bundle-size/).

Reviewer: ```suggestion
  Check the updated [Supported platforms documentation](https://next.material-ui.com/getting-started/supported-platforms/) and [new "minimizing bundle size" guide](https://next.material-ui.com/guides/minimizing-bundle-size/) for more details.
```

>   If you target IE11, you need to use the new bundle (`legacy`). We are treating IE11 as a second class-citizen, it's a continuation of the direction taken in #22873.

Reviewer: ```suggestion
  If you target IE11, you need to use the new bundle (`legacy`). We are treating IE11 as a second class-citizen, which is a continuation of the direction taken in #22873.
```

- 🚀 Improve the internal benchmark suite (#22923, #23058) @mnajdova.

>   This was a prerequisite step to improve the system. We needed to be able to measure performance. After #22945, we have measured that the `Box` component is x3 faster on v5-alpha compared to v4.

Reviewer: ```suggestion
  This was a prerequisite step to improve the [system](https://material-ui.com/system/basics/). We needed to be able to measure performance. After #22945, we have measured that the `Box` component is x3 faster in v5-alpha compared to v4.
```

- ✏️ A new blog post: [Q3 2020 Update](https://material-ui.com/blog/2020-q3-update/) (#23055) @oliviertassinari.

> - 🐙 Migrate more test to react-testing-library @deiga, @Morteza-Jenabzadeh, @nicholas-l.

Reviewer: ```suggestion
- 🐙 Migrate more tests to react-testing-library @deiga, @Morteza-Jenabzadeh, @nicholas-l.
```
